src/Microsoft.Management.Configuration/Microsoft.Management.Configuration.idl

```diff
@@ -715,6 +715,12 @@ namespace Microsoft.Management.Configuration
 
         // Creates a processor from the given factory.
         ConfigurationProcessor CreateConfigurationProcessor(IConfigurationSetProcessorFactory factory);
+
+        // Whether configuration is enabled.
+        Boolean IsConfigurationEnabled{ get; };
+
+        // Enables configuration. Requires store access.
+        Windows.Foundation.IAsyncActionWithProgress<UInt32> EnableConfigurationAsync();

```

Comment:
```suggestion
        Boolean IsConfigurationAvailable{ get; };

        // Enables configuration. Requires store access.
        Windows.Foundation.IAsyncActionWithProgress<UInt32> EnsureConfigurationAvailableAsync();
```
If there is a policy that disables configuration, I feel like the name of this becomes confusing.  A word that invokes "present" seems better.